cartographer/mapping/sparse_pose_graph.h

```diff
@@ -106,9 +106,9 @@ class SparsePoseGraph {
   virtual std::vector<transform::Rigid3d> GetSubmapTransforms(
       const Submaps& submaps) = 0;
 
-  // Returns the transform converting from odometry based transforms to the
-  // optimized world.
-  virtual transform::Rigid3d GetOdometryToMapTransform(
+  // Returns the transform converting from local SLAM based transforms to the
```

Comment:
Returns the transform converting data in the local map frame (i.e. the continuous, non-loop-closed frame) to the global map frame (i.e. the discontinuous, loop-closed frame).